### PR TITLE
New CMO type hardsuit

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
[Changelogs]
Again didnt touch sprites do to tauric - Tested and works Poojawa  
Adds a new CMO hardsuits -
This hard suit has less armor then normal medical suits for combat but is shielded more against rads and acid
The hardsuit has no slow down modifier on it so you can quickly get to the dieing person and quickly get them out
The CMO hardsuit has a built in Med-Hud as well as flash protection. As well as a expanded use for its exo suit storage. 
CMO hardsuit comes in a suit storage with a double expanded small o2 tank
***THIS IS NOT MEANT AT ALL FOR COMBAT***
:cl: optional name here
add: CMO hardsuit
/:cl:

[why]
CMO sharing a shitty medical suit dosnt really seem fair to the other heads of staff that get a upgraded fancy suit.
CMOs are ment to be fast and able to heal the wounded, so thats the reason for the speed
I do *not* want this suit to be used in combat. It is no intention for a combat CMO to meme use this suit as a few armor set, hints the lower armor values

### CODE REVIEW

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 1]
```diff
@@ -457,6 +449,41 @@
...
+	tauric = TRUE		//Citadel Add for tauric hardsuits
+
+/obj/item/clothing/head/helmet/space/hardsuit/medical/CMO
+	name = "CMO hardsuit helmet"
+	desc = "A Very.Med special helmet designed for work in a hazardous, low pressure environment. Built with ultra-lightweight materials for extra comfort and movement, comes with bult in eye protection and a medical hud."
+	icon_state = "hardsuit0-medical"
+	item_state = "medical_helm"
+	item_color = "medical"
+	flash_protect = 1
+	armor = list("melee" = -5, "bullet" = 0, "laser" = -15, "energy" = -15, "bomb" = 15, "bio" = 120, "rad" = 80, "fire" = 90, "acid" = 95)
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/spacesuits/hardsuit.dm]: Less bomb for head do to huds/flash protection - As well as not wanting this used in combat

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 2]
```diff
@@ -457,6 +449,41 @@
...
+	species_exception = list(/datum/species/angel)
+	tauric = TRUE		//Citadel Add for tauric hardsuits
+
+/obj/item/clothing/head/helmet/space/hardsuit/medical/CMO
+	name = "CMO hardsuit helmet"
+	desc = "A Very.Med special helmet designed for work in a hazardous, low pressure environment. Built with ultra-lightweight materials for extra comfort and movement, comes with bult in eye protection and a medical hud."
+	icon_state = "hardsuit0-medical"
+	item_state = "medical_helm"
+	item_color = "medical"
+	flash_protect = 1
```
Comment (Trilbyspaceclone) [inline code/modules/clothing/spacesuits/hardsuit.dm:470]: Flash protection against things like small bombs is the idea. Saving wounded well under fire, ect ect

**Comment by Poojawa** [comment]
what is with the space suit creep now. ree

File: code/modules/clothing/spacesuits/hardsuit.dm [hunk 3]
```diff
@@ -457,6 +449,41 @@
...
+	icon_state = "hardsuit-medical"
+	name = "CMO medical hardsuit"
+	desc = "A Very.Med special suit that protects against hazardous, low pressure environments. Built with ultra-lightweight materials for easier movement. Has less armor then the normal Medical hardsuit but allows for greater mobility"
+	item_state = "medical_hardsuit"
+	allowed = list(/obj/item/flashlight, /obj/item/tank/internals, /obj/item/storage/firstaid, /obj/item/healthanalyzer, /obj/item/stack/medical, /obj/item/defibrillator, /obj/item/storage/belt/medical, /obj/item/reagent_containers/hypospray, /obj/item/pinpointer/crew, /obj/item/sensor_device)
+	armor = list("melee" = 5, "bullet" = 0, "laser" = -10, "energy" = -5, "bomb" = 35, "bio" = 120, "rad" = 80, "fire" = 90, "acid" = 95)
+	helmettype = /obj/item/clothing/head/helmet/space/hardsuit/medical/CMO
+	slowdown = 0 //So CMO can move with easy.
+	species_exception = list(/datum/species/angel)
+	tauric = TRUE		//Citadel Add for tauric hardsuits
```
Comment (Poojawa) [inline code/modules/clothing/spacesuits/hardsuit.dm]: unneeded, it's a child object of the medical suit

File: code/modules/clothing/spacesuits/hardsuit.dm (hunk 1 above)
Comment (Poojawa) [reply code/modules/clothing/spacesuits/hardsuit.dm]: Medical staff already have a reagent scanner. This is functionally worse in every way except for the speed change and flashbang cheese.

**Review by Poojawa — CHANGES_REQUESTED** [review]
it's just... a weird tangent you're doing lately in giving all hardsuits built in specialized huds. And weirder still are your numbers. for armor value.

You do realize that -15 is 15% bonus damage taken, right? You're more armored being completely naked. I really don't agree with these changes, at all for the 'advantage' of slightly faster speed and flash protect.

File: code/modules/clothing/spacesuits/hardsuit.dm (hunk 1 above)
Comment (Trilbyspaceclone) [reply code/modules/clothing/spacesuits/hardsuit.dm]: The reagent scanner is apart of normal medical suits

**Comment by Trilbyspaceclone** [comment]
@Poojawa I know. Its not meant for combat, CMO doesn't normally get armor.
Its also to stop people from - `Ima always ware this no slow down hard suit for pwer gamer reasons`

**Comment by Poojawa** [comment]
so just zero it out. there's no logic in making them take ***more*** damage

**Comment by silicons** [comment]
ehh let's not full speed hardsuit.

**Pull request closed without merging**